### PR TITLE
docs: Publish v9.0.0-alpha.0 release highlights

### PR DESCRIPTION
**Pull request by mdjermanovic**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What changes did you make? (Give an overview)

Added highlights and published the  v9.0.0-alpha.0 release blog post.

### CODE REVIEW

File: src/content/blog/2023-12-29-eslint-v9.0.0-alpha.0-released.md [hunk 1]
```diff
@@ -3,21 +3,78 @@ layout: post
...
+Most of the highlights of this release are breaking changes, and are discussed further in the [migration guide](/docs/next/use/migrate-to-9.0.0). There are summaries of the significant changes below. (Less significant changes are included in the migration guide.)
 
-## Highlights
+### Node.js < v18.18.0, v19 no longer supported
+
+As of this post, Node.js v20.x is the LTS release, and as such we are [dropping support](https://github.com/eslint/eslint/issues/17595) for all versions of Node.js prior to v18.18.0 as well as v19.x.
+
+### Flat config now the default and has some changes
+
+Flat config is now the default configuration format for ESLint and eslintrc is officially deprecated. beginning in v9.0.0 and eslintrc will be deprecated but will still work. To continue using a eslintrc configuration file, you'll need to set the `ESLINT_USE_FLAT_CONFIG` environment variable to `false`.
```
Comment (mdjermanovic) [inline src/content/blog/2023-12-29-eslint-v9.0.0-alpha.0-released.md:27]: `beginning in v9.0.0 and eslintrc will be deprecated but will still work.` looks like a copy-paste error.

Comment (nzakas) [reply src/content/blog/2023-12-29-eslint-v9.0.0-alpha.0-released.md:27]: Good catch

**Comment by mdjermanovic** [comment]
This shouldn't be a link:

image

**Comment by nzakas** [comment]
Sadly, no idea how to disable that. It's automatically linked by Eleventy.

**Comment by mdjermanovic** [comment]
Ok, I just removed `.md`.

**Pull request merged**